src/modules/site-v2/base/views/about/about.py

```diff
@@ -31,7 +31,7 @@ def about():
   ''' About us Page - Gives an overview of CaeNDR '''
   title = "About CaeNDR"
   disable_parent_breadcrumb = True
-  isotypes = get_isotypes(known_origin=True)
+  isotypes = get_isotypes(known_origin=True, species='c_elegans')

```

Comment:
@vlagrassa -- based on our earlier conversation, let's move towards removing all strings as identifiers from the code base and instead use "enums" or class-enums. See the caendr.model for `TaskStatus` as an example.

Reply:
The two static `"c_elegans"` references in this file are actually unnecessary -- removed in 1df7d83b8c2c915cccfd8fd9ad1b296727b39c5b.

A few more static references remain in the codebase, e.g. in the Downloads workflow.  Since we'll have to overhaul this page anyways, I'm holding off on it for now.